src/database/seeders/20220302143415-demo-user.js

```diff
@@ -1,32 +1,25 @@
+const roles = require('../../helpers/roles');
+
 module.exports = {
   async up(queryInterface, Sequelize) {
-    return queryInterface.bulkInsert('Users', [
+    return queryInterface.bulkInsert('Roles', [
       {
-        firstName: 'John',
-        lastName: 'Doe',
-        email: 'example@example.com',
-        userName: 'testUser1',
-        password: 'password',
-        facebookId: '11111111113333',
+        name: roles.default.REQUESTER,
         createdAt: new Date(),
         updatedAt: new Date(),
       },
       {
-        firstName: 'andela',
-        lastName: 'kigali',
-        email: 'andela@andela.com',
-        userName: 'testUser2',
-        password: 'password2',
+        name: roles.default.MANAGER,
         createdAt: new Date(),
         updatedAt: new Date(),
       },
       {
-        userName: 'Johnny',
-        email: 'vwlixyakslqavbonkw@kvhrw.com',
-        password:
-          '$2b$10$bQUIGi5mWZ8NRFMN3.VjgOiV1fPTYExzxwhOpL.m/bgD6tiLS2hxm',
-        firstName: 'John',
-        lastName: 'Doe',
+        name: roles.default.ADMIN,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: roles.default.SUPER_ADMIN,
```

Comment:
do not update existing seeds just as models, create new ones and do the changes to pre-existing data if you have to